packages/dart/sshnoports/bundles/shell/headless/srvd.sh

```diff
@@ -1,20 +1,16 @@
 #!/bin/sh
 # disable "var is referenced but not assigned" warning for template
 # shellcheck disable=SC2154
-
-# Configuration of srvd service
-# This unit script is a template for the srvd background service.
-# You can configure the service by editing the variables below.
-# This service file covers the common configuration options for srvd.
-# To see all available options, run `srvd` with no arguments.
-
+# SCRIPT METADATA
+binary_path="$HOME/.local/bin"
```

Comment:
Did this so we could remove the separate root version, by simply replacing the string here